src/it/scala/temple/containers/DockerPostgresService.scala

```diff
@@ -10,7 +10,7 @@ import scala.concurrent.duration._
 
 /** DockerPostgresService encapsulates all configuration for running a postgres docker container */
 object DockerPostgresService {
-  val image            = "postgres:9.6"
+  val image            = "postgres:12.1"
```

Comment:
Can you try and break your pull requests into smaller changes in future, e.g. to 10.11, then another to 11.6, then to 12.1?